cmake/modules/FindFluidSynth.cmake

```diff
@@ -22,7 +22,7 @@ find_path(FluidSynth_INCLUDE_DIR
 )
 
 find_library(FluidSynth_LIBRARY
-	NAMES "fluidsynth"
+	NAMES "fluidsynth" "fluidsynth-3" "fluidsynth-2" "fluidsynth-1"
```

Comment:
Out of morbid curiosity... can you explain what lead to this fix?

Reply:
On Windows, the fluidsynth library is named "fluidsynth-3" rather than just "fluidsynth". I think this is because the Linux version uses ".so.3" so they did it that way as the Windows equivalent. Something must have changed in vcpkg to cause it to not work with just "fluidsynth" anymore.

Reply:
Looks like "fluidsynth-2" and "fluidsynth-1" are unnecessary so I'll remove them